packages/combo-box/test/internal-filtering.test.js

```diff
@@ -260,7 +259,6 @@ describe('internal filtering', () => {
     it('should properly display all items in the selector', () => {
       comboBox.open();
       comboBox.filteredItems = makeItems(10);
-      flush();
```

Comment:
I'm under impression that  `flush()` was historically imported from the wrong place. It should have rather been imported from `untils/render-status/flush.js` because there is a `afterNextRender` call that takes place when the overlay is opening. It would be reasonable to ensure the overlay has been completely opened = all the corresponding logic has been executed.